app/Services/EdgeProxyRemoteRouteService.php

```
@@ -0,0 +1,1562 @@
+
+        if ($host === '') {
+            return null;
+        }
+
+        if (filter_var($host, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6)) {
+            return '['.$host.']';
+        }
+
+        return $host;
+    }
+}
```

PHPMD (2.15.0)

[warning] The class EdgeProxyRemoteRouteService has 1547 lines of code. Current threshold is 1000. Avoid really long classes. (undefined)

(ExcessiveClassLength)